src/tests/ftest/util/test_utils_pool.py

```diff
@@ -101,7 +101,7 @@ def get_params(self, test):
 
         # Autosize any size/scm_size/nvme_size parameters
         # pylint: disable=too-many-boolean-expressions
-        if ((self.size.value is not None and str(self.size.value).endswith("%"))
+        if ((self.size.value is not None)
                 or (self.scm_size.value is not None
```

Comment:
The the intended change we need to check `self.size` at all so we can extend the modifications to this line:
```suggestion
        if ((self.scm_size.value is not None
```

Note: there is one very strange situation (in that it doesn't make sense to include these two parameters together - and hopefully the command would bark at that) where we could still end up replacing a `self.scm_size.value` ending with a `%` with a calculated value.  If both a `self.scm_size.value` and either a `self.scm_size.value` and/or a `self.nvme_size.value` are defined in the test yaml, then we still pass `self.scm_size.value` into the `autosize_pool_params()` method call on line 111-117.  We could avoid this by passing `tier_ratio=None, scm_size=None,` instead of the actual values.

Reply:
I think a good followup PR would be to replace all uses of `scm_size: %` and `nvme_size: %` with `size: %` and `tier_ratio`, and remove `autosize_pool_params()` altogether. Thoughts?

Reply:
That sounds good. Although we can't use `size: %` and `tier_ratio` together. If we use them together, `tier_ratio` is ignored.

Reply:
Do you mean `tier_ratio` is ignored by `dmg pool create`? That seems like a bug! :)

Reply:
Removed the self.size check.

Reply:
I think my point might have been missed.  I think we should also be making this line change on lines 110-116 in https://github.com/daos-stack/daos/blob/fd5b499b954153078ce6f96b6b11ec73350d76d5/src/tests/ftest/util/test_utils_pool.py:
```
                params = test.server_managers[index].autosize_pool_params(
                    size=None,
                    tier_ratio=None,
                    scm_size=self.scm_size.value,
                    nvme_size=self.nvme_size.value,
                    min_targets=self.min_targets.value,
                    quantity=self.quantity.value)
```

Reply:
Thanks. Now I understand what you meant.